techniques/general/G226.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Providing audio descriptions by incorporating narration in the soundtrack</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/></head><body><h1>Providing audio descriptions by incorporating narration in the soundtrack</h1><section class="meta"><p class="id">ID: G226</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section><section id="applicability"><h2>When to Use</h2>
+      <p>Any technology that supports audio and video.</p>
+   </section><section id="description"><h2>Description</h2>
+      <p>The objective of this technique is to provide audio descriptions through  narrative incorporated into the soundtrack of the synchronized video, so that people who cannot see are able to understand important visual material.</p>
+      <p>Since most user agents today cannot merge multiple sound tracks, this technique adds additional context by revising the draft or pre-existing soundtrack so that the narrative includes audio description via a single audio track. This additional information may address actions, characters, scene changes, and on-screen text (not captions) that are important to understanding the content.</p>
```

Comment:
```suggestion
      <p>Since most user agents today cannot merge multiple sound tracks, this technique adds additional context by revising the draft or pre-existing soundtrack so that the narration includes audio description via a single audio track. This additional information may address actions, characters, scene changes, and on-screen text (not captions) that are important to understanding the content.</p>
```